CHANGELOG.unreleased.md

```diff
@@ -38,6 +38,7 @@
 - @vates/nbd-client major
 - @vates/node-vsphere-soap major
 - @xen-orchestra/backups minor
+- @xen-orchestra/fs patch
```

Comment:
My bad, I misunderstand, it does not need to be release, because it's a test fix!